bin/main.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:http/http.dart' as http;
+
+Future<String> download(final url) async {
+  final response = await http.get(url);
+  dynamic htmlToParse = '';
+  // If the http request is successful the statusCode will be 200
+  if (response.statusCode == 200) {
+    htmlToParse = response.body;
```

Comment:
Instead of declaring a `var` variable up there, assigning it only one time, then returning it, prefer returning the value directly:

```dart
if (... == 200) {
  final body = response.body;
  return body;
}
return '';
```

Reply:
The advantage is you can use `final` instead of `var`. It's not that important here, but on the long run, using `final` as much as possible will make your code more readable and help you avoid bugs.